daisy_workflows/image_import/windows/translate_bootstrap.ps1

```diff
@@ -160,8 +160,8 @@ try {
   }
 
   $driver_dir = 'c:\drivers'
-  New-Item $driver_dir -Type Directory | Out-Null
-  New-Item $script:components_dir -Type Directory | Out-Null
+  New-Item $driver_dir -Force -Type Directory | Out-Null
+  New-Item $script:components_dir -Force -Type Directory | Out-Null
```

Comment:
Also, this directory shouldn't exist prior to import.

Reply:
What about images that were exported from GCP and are being re-imported?

Reply:
This command is running on the import VM, using a GCP image.

Reply:
Same as above. If it's harmless, I prefer to add -Force, because a pre-exist folder name shouldn't block the translation in any case.

Reply:
My understanding is that it's running on worker VM but is creating directories on the imported image. Is that a correct assumption?

If so, these directories might exist for an image that is being re-imported.

Reply:
No, your assumption is incorrect. The driver and components directories are created on the worker VM - not the imported image.

I believe these two statements should not contain -Force, but I understand the premise of this PR and not failing when a image in reimported to GCP on more than once.

Reply:
removed these 2 lines